libs/fixtures/src/data/electionFamousNames2021/index.ts

```diff
@@ -1,4 +1,59 @@
+import { Election, GridLayout } from '@votingworks/types';
+import { election } from './election.json';
+import { asElectionDefinition } from '../../util';
+
 export { election, electionDefinition } from './election.json';
 export * as electionJson from './election.json';
 export * as machineMarkedBallotPage1 from './bmd-ballot-complete-p1.jpg';
 export * as machineMarkedBallotPage2 from './bmd-ballot-complete-p2.jpg';
+
+const partialMockGridPosition = {
+  type: 'option',
+  side: 'front',
+  column: 0,
+  row: 0,
+  contestId: 'mayor',
+  optionId: 'sherlock-holmes',
+} as const;
+
+const mockMultiSheetGridLayouts: GridLayout[] = [
+  {
+    ballotStyleId: '1',
+    optionBoundsFromTargetMark: {
+      left: 0,
+      right: 0,
+      top: 0,
+      bottom: 0,
+    },
+    gridPositions: [
+      {
+        ...partialMockGridPosition,
+        sheetNumber: 1,
+      },
+      {
+        ...partialMockGridPosition,
+        sheetNumber: 2,
+      },
+      {
+        ...partialMockGridPosition,
+        sheetNumber: 3,
+      },
+    ],
+  },
+];
+
+/**
+ * Election with mock multi-sheet grid layouts. The layouts are invalid and
+ * cannot be used for scanning, but the data structure is useful for testing.
+ */
+export const multiSheetElection: Election = {
+  ...election,
+  gridLayouts: mockMultiSheetGridLayouts,
+};
```

Comment:
Could we use the existing HMPB fixtures for this? I think it's prudent to try to avoid proliferation of different election fixtures

Reply:
We need a multi-sheet fixture that can be used in both the frontend and the backend - would we want to import those fixtures from `hmpb/render-backend` into frontends? I also think this fixture is pretty low maintenance, since it's just based off of another fixture.

Reply:
Most of those ballot fixtures are multi-sheet.

I also don't think there's a point in maintaining `electionFamousNames` as a non-grid-layouts fixture. I guess what I'm ultimately recommending is that we convert all of our base fixtures to the VxDesign-exported election package.

Reply:
But that's not in scope for this PR

Reply:
Are you sure the fixtures are mostly multi-sheet? If I search the repo for `"sheetNumber": 1` I get thousands of hits, but if I search the repo for `"sheetNumber": 2` I get zero hits.

Reply:
I think it's because the elections are serialized in CDF. Try `"Sheet": 2`

Reply:
Oh, got it.

Still wondering about my first question - is it OK to import fixtures from `hmpb/render-backend` into the frontends?

Reply:
Good question. I think if it's just for tests it should be fine, right?

Reply:
For tests it should be fine, but for frontend tests there may be some issues. There are a couple of things to polyfill or stub. I just pushed up a commit with my understanding of how you want this to work. The issue I haven't worked out is how to make it work with `storybook`. It doesn't work with the fixtures from `render-backend` because I think it is trying to import two different copies of `react`, which causes problems.

Reply:
Oh shoot that's annoying, I didn't think about the issues with importing into a frontend test environment. I think it's nice to just avoid importing backend packages into frontend environments rather than have a stub/polyfill house of cards. That makes me think maybe we need to find a way to move the fixture data into a different package that doesn't have the actual Node code that generates the fixtures. What do you think about that? It also doesn't have to happen in this PR - I'm ok with either keeping the polyfill approach you have now or abandoning ship for the moment and going back to what you had originally.